src/utils/macros.rs

```diff
@@ -201,3 +201,37 @@ macro_rules! impl_from {
         }
     };
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::utils::command::Command;
+    #[cfg(windows)]
+    use crate::utils::error::ErrorKind;
+    use std::io::{stdout, Write};
+    pub struct FakeCommand;
```

Comment:
Also, please reorder imports in the following way and add an empty line before `FakeCommand`.

```rust
use std::io::{stdout, Write};

use crate::utils::command::Command;
#[cfg(windows)]
use crate::utils::error::ErrorKind;

pub struct FakeCommand;
```